tests/foreman/api/test_http_proxy.py

```diff
@@ -140,3 +147,27 @@ def test_positive_end_to_end(function_http_proxy, target_sat, module_manifest_or
     repo_2.http_proxy_policy = 'none'
     repo_2.update(['http_proxy_policy'])
     assert repo_2.http_proxy_policy == 'none'
+
+    # test scenario for yum type repo discovery.
+    repo_name = 'fakerepo01'
+    yum_repo = target_sat.api.Organization(id=module_manifest_org.id).repo_discover(
+        data={
+            "id": module_manifest_org.id,
+            "url": settings.repos.repo_discovery.url,
+            "content_type": "yum",
+        }
+    )
+    assert len(yum_repo['output']) == 1
+    assert yum_repo['output'][0] == f'{settings.repos.repo_discovery.url}/{repo_name}/'
+
+    # test scenario for docker type repo discovery.
+    yum_repo = target_sat.api.Organization(id=module_manifest_org.id).repo_discover(
+        data={
+            "id": module_manifest_org.id,
+            "url": 'quay.io',
```

Comment:
Non-blocking but this still works without having `https://` in the url?

Reply:
yes, it works.